### PR TITLE
Add contact page

### PR DESCRIPTION
**Pull request by benjy**

### CODE REVIEW

**Comment by Sam152** [comment]
Lets add another menu item to the default content to link to the contact page.

**Comment by benjy** [comment]
test this please

File: agov/src/Tests/ContactPageTest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->adminUser = $this->drupalCreateUser([]);
+  }
+
+  /**
+   * Test we're on the configurable dependencies.
```
Comment (Sam152) [inline agov/src/Tests/ContactPageTest.php]: Incorrect comment?

File: agov/src/Tests/ContactPageTest.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    // Anonymous users can view the contact page and get the name fields.
+    $this->drupalGet('/contact');
+    $this->assertResponse(200);
+    $this->assertFieldById('edit-message-0-value');
+
+    // Authenticated users get the form as well.
+    $this->drupalLogin($this->adminUser);
+    $this->drupalGet('/contact');
+    $this->assertResponse(200);
+    $this->assertFieldById('edit-message-0-value');
```
Comment (Sam152) [inline agov/src/Tests/ContactPageTest.php]: Why not test a submission while we're here?

**Comment by Sam152** [comment]
Looking good after feedback resolved.

File: agov/src/Tests/AgovTestBase.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace Drupal\agov\Tests;
+
+use Drupal\simpletest\WebTestBase;
+use Symfony\Component\BrowserKit\Request;
+
+/**
+ * Base class for testing aGov.
+ */
+abstract class AgovTestBase extends WebTestBase {
```
Comment (jibran) [inline agov/src/Tests/AgovTestBase.php]: Can we use `BrowserTestBase`?

**Pull request merged**